images/metering-ansible-operator/roles/meteringconfig/tasks/configure_networking.yml

```diff
@@ -71,7 +71,7 @@
     _ocp_enabled_global_proxy_https_proxy: "{{ _cluster_proxy_config.status.httpsProxy }}"
   no_log: true
   when:
-  - cluster_proxy_config is defined
+  - cluster_proxy.config.resources is defined
```

Comment:
It's not redundant to check if `defined` and `length > 0`

Reply:
At a glance it's redundant - I know this type of pattern, i.e. check if the dictionary key exists and check if non-empty, was required in previous Ansible versions. I don't think there's been any updates to the `length` jinja2 filter that handles the failure case where that dictionary key isn't defined.

Reply:
ah, okay.